lib/apr/views/ach_slack_view.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Apr.Views.ACHSlackView do
+  import Apr.Views.Helper
+
+  def render(subscription, event = %{"verb" => "dispute_created",}, _) do
+    case {subscription.theme, event["verb"], event["properties"]} do
+      {"ach", "dispute_created", %{"payment_method" => "us_bank_account"}} ->
+        generate_slack_message(event, "dispute_created")
+      _ -> nil
+    end
+  end
+
+  def render(_, _, _), do: nil
+
+  defp generate_slack_message(event, _routing_key) do
+    order_id = event["object"]["id"]
+    seller_id = event["properties"]["seller_id"]
+    seller_path = "partner/#{seller_id}"
+    payment_intent_id = event["properties"]["payment_intent_id"]
+    payment_intent_link = "https://dashboard.stripe.com/payments/#{payment_intent_id}"
+
+    %{
+      text: ":alert: Dispute, do not refund this order on Stripe",
+      attachments: [
+        %{
+          fields: [
+            %{
+              title: "Order ID",
+              value: "<#{exchange_admin_link(order_id)}|#{order_id}>",
+              short: true
+            },
+            %{
+              title: "Seller ID",
+              value: "<#{admin_partners_link(seller_path)}|#{seller_id}>",
+              short: true
+            },
+            %{
+              title: "Stripe payment ID",
+              value: "<#{payment_intent_link}|#{payment_intent_id}>",
```

Comment:
`payment_intent_id` is added in this PR: https://github.com/artsy/exchange/pull/1311#issuecomment-1219459394